File: CS/DevExpress.Upload/LocalHelper.cs
using System;
using System.IO;
using System.Linq;
using System.Web;
using DevExpress.Upload.Properties;

namespace DevExpress.Upload {
    internal static class LocalHelper {

        internal static String ReadFileAsString(String name) {
            using(var file = new StreamReader(ResolveFilePath(name))) {
                var text = file.ReadToEnd();
                return text;
            }
        }

        internal static Byte[] ReadFileAsBytes(String name) {
            using(var file = new FileStream(ResolveFilePath(name), FileMode.Open)) {
                var bytes = new Byte[file.Length];
                file.Read(bytes, 0, bytes.Length);
                return bytes;
            }
        }

        internal static FileType GetFileType(String name) {
            var ext = Path.GetExtension(name);
            if((new[] { ".html", ".htm" }).Contains(ext))
                return FileType.Html;
            if((new[] { ".txt" }).Contains(ext))
                return FileType.Text;
            if((new[] { ".jpg", ".jpeg", ".jpe" }).Contains(ext))
                return FileType.Image;
            return FileType.Unknown;
        }

        internal static void SaveToFile(String value, String name) {
            using(var file = new StreamWriter(ResolveFilePath(name), false)) {
                file.Write(value);
            }
        }

        internal static String ResolveFilePath(String name) {
            return HttpContext.Current.Server.MapPath(Settings.Default.StoragePath) + name;
        }
    }
}

File: VB/DevExpress.Upload/LocalHelper.vb
Imports System
Imports System.IO
Imports System.Linq
Imports System.Web

Namespace DevExpress.Upload
    Friend NotInheritable Class LocalHelper

        Private Sub New()
        End Sub


        Friend Shared Function ReadFileAsString(ByVal name As String) As String
            Using file = New StreamReader(ResolveFilePath(name))
                Dim text = file.ReadToEnd()
                Return text
            End Using
        End Function

        Friend Shared Function ReadFileAsBytes(ByVal name As String) As Byte()
            Using file = New FileStream(ResolveFilePath(name), FileMode.Open)
                Dim bytes = New Byte(file.Length - 1){}
                file.Read(bytes, 0, bytes.Length)
                Return bytes
            End Using
        End Function

        Friend Shared Function GetFileType(ByVal name As String) As FileType
            Dim ext = Path.GetExtension(name)
            If ( { ".html", ".htm" }).Contains(ext) Then
                Return FileType.Html
            End If
            If ( { ".txt" }).Contains(ext) Then
                Return FileType.Text
            End If
            If ( { ".jpg", ".jpeg", ".jpe" }).Contains(ext) Then
                Return FileType.Image
            End If
            Return FileType.Unknown
        End Function

        Friend Shared Sub SaveToFile(ByVal value As String, ByVal name As String)
            Using file = New StreamWriter(ResolveFilePath(name), False)
                file.Write(value)
            End Using
        End Sub

        Friend Shared Function ResolveFilePath(ByVal name As String) As String
            Return HttpContext.Current.Server.MapPath(My.Settings.Default.StoragePath) & name
        End Function
    End Class
End Namespace